lib/stat_tracker.rb

```diff
@@ -1,84 +1,89 @@
 require_relative './games_collection'
 require_relative './teams_collection'
 require_relative './game_teams_collection'
+require 'csv'
 
+#THE PARENT
 class StatTracker
-  attr_reader :games_collection, :teams_collection, :game_teams_collection
```

Comment:
We'll have to be really careful how we're using games_collection vs games so we're always clear what we're talking about too. Maybe we need to think of some more colorful names for things to help differentiate them